eng/Versions.props

```diff
@@ -54,9 +44,17 @@
     <NUnit3TemplatesVersion>1.6.1</NUnit3TemplatesVersion>
   </PropertyGroup>
   <PropertyGroup>
+    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview5-27607-08</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreDotNetHostResolverPackageVersion>$(MicrosoftNETCoreAppPackageVersion)</MicrosoftNETCoreDotNetHostResolverPackageVersion>
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
     <NetCoreAppTargetingPackVersion>$(MicrosoftNETCoreAppPackageVersion)</NetCoreAppTargetingPackVersion>
+    <NetStandardPatchSeparatorIndex>$([MSBuild]::Add($(MicrosoftNETCoreAppPackageVersion.IndexOf('.')), 1))</NetStandardPatchSeparatorIndex>
+    <NetStandardPatchSeparatorIndex>$(MicrosoftNETCoreAppPackageVersion.IndexOf('.', $(NetStandardPatchSeparatorIndex)))</NetStandardPatchSeparatorIndex>
+    <NetStandardTargetingPackSuffix>$(MicrosoftNETCoreAppPackageVersion.Substring($(NetStandardPatchSeparatorIndex)))</NetStandardTargetingPackSuffix>
```

Comment:
I don't understand why we are mixing netstandard and netcoreapp package versions together. The netstandard targeting pack should be allowed to have its own version.

Reply:
I agree, and maybe some extra motivation: the netstandard targeting pack build will be moved to dotnet/standard at some point, so it will be necessary for it to have a different version.